### PR TITLE
Manifests template updates

### PR DESCRIPTION
**Pull request by ciiay**
This PR is for creating the right manifests and image of gitops-plugin.

### CODE REVIEW

File: src/components/ApplicationListPage.tsx [hunk 1]
```diff
@@ -9,10 +9,11 @@ import {
 } from '@openshift-console/dynamic-plugin-sdk';
 import { LoadingBox } from '@patternfly/quickstarts';
 
+import { fetchDataFrequency, pipelinesBaseURI } from '../const';
```
Comment (keithchong) [inline src/components/ApplicationListPage.tsx:12]: Do you have extra commits here?  Looks like these changes in the src folder have already been merged.

File: manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    matchLabels:
+      app: gitops-plugin
+  template:
+    metadata:
+      labels:
+        app: gitops-plugin
+    spec:
+      containers:
+        - name: gitops-plugin
+          image: quay.io/yicai/console-gitops-plugin:v0.0.2
```
Comment (keithchong) [inline manifest.yaml:24]: Do we have a redhat-developer image on quay.io? (If not, I suppose we can change this later when it's ready)

File: manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,112 @@
+apiVersion: apps/v1
```
Comment (keithchong) [inline manifest.yaml:1]: Question: how did you test this specific manifest?  Which OpenShift cluster version(s) did you apply this manifest to, and did you check that the plugin is enabled and were our pages populated?  Take note that we should document this in the readme as part of the documentation story.

File: manifest.yaml (hunk 2 above)
Comment (ciiay) [reply manifest.yaml]: Updated with this new quay.io repo url: quay.io/redhat-developer/gitops-console-plugin

File: src/components/details/ResourcesSection.tsx [hunk 4]
```diff
@@ -129,7 +129,7 @@ const GitOpsResourcesSection: React.FC<GitOpsResourcesSectionProps> = ({
                 <SplitItem>
                   <Stack style={{ marginRight: 'var(--pf-global--spacer--sm)' }}>
                     {resourcesList.map((eachResource) => (
-                      <StackItem>
+                      <StackItem key={eachResource.name}>
```
Comment (keithchong) [inline src/components/details/ResourcesSection.tsx:132]: Why did you use the name as the key here whereas the kind is used in the other two places?

Comment (keithchong) [reply src/components/details/ResourcesSection.tsx:132]: Otherwise, LGTM

**Review by keithchong — APPROVED** [review]
LGTM

**Pull request merged**